logger.go

```diff
@@ -129,3 +158,38 @@ func (l *MicroLogger) processStack(keyVals []interface{}) []interface{} {
 
 	return keyVals
 }
+
+func (l *CtxMicroLogger) Debug(format string, params ...interface{}) {
```

Comment:
I figure `Info` is quite rare although not non-existing. `Warning` is used quite a bit in `azure-operator` because there are situations where operator doesn't fail as such but Azure is returning usual errors while it cannot keep up, but I guess those can be put to `Debug` level as well since the operator will eventually reconcile the situation and we never do anything about the log level besides some coloring which is mostly just important for `Error`.

I'll reduce for `Debug` and `Error` and see how it goes.